ecl/eclagent/eclagent.cpp

```diff
@@ -1864,6 +1864,7 @@ void EclAgent::doProcess()
         {
             MTIME_SECTION(queryActiveTimer(), "Process");
             Owned<IEclProcess> process = loadProcess();
+            QueryTerminationCleanup threadCleanup;
```

Comment:
As far as I understand it, this isn't actually going to used, i.e. there won't be a thread termination hook installed on this thread..
That confused me when reviewing this PR a bit.
In Roxie the equivalent code is on it's own thread

@ghalliday - correct me if I'm wrong.